mapfishapp/src/docker/Dockerfile

```diff
@@ -4,6 +4,13 @@ ADD . /
 
 RUN java -jar "$JETTY_HOME/start.jar" --add-to-startd=jmx,jmx-remote,stats
 
+RUN apt-get update && \
+   apt-get install -y libgdal1h libgdal-java gdal-bin && \
```

Comment:
`gdal-bin` depends on `libgdal1h` : https://packages.debian.org/jessie/gdal-bin
So you can remove `libgdal1h`